### PR TITLE
SqlAssessment version 0.6.1

### PR DESCRIPTION
**Pull request by vchvlad**
SqlAssessment version 0.6.1

### CODE REVIEW

**Comment by vchvlad** [comment]
@Charles-Gagnon Could you please update blob with 0.6.1 version? Thank you.

File: extensionsGallery.json [hunk 1]
```diff
@@ -3233,7 +3233,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.26.0"
+									"value": ">=1.27.0"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:3236]: We haven't released 1.27 yet so this isn't a valid engine for the stable gallery. Remove the update to that file from this PR and then submit a new PR against the [extensions/RC1](https://github.com/microsoft/azuredatastudio/tree/extensions/rc1) branch for the stable gallery update as per https://github.com/microsoft/azuredatastudio-docs/blob/main/Guides/UpdateExtensionGallery.md#scenarios

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

**Comment by vchvlad** [comment]
@Charles-Gagnon Thanks for updating rc1 gallery. I guess now insiders gallery can be updated as well, right?

**Comment by Charles-Gagnon** [comment]
Thanks for the reminder! Gallery updated

**Pull request merged**